integration_tests/tests/in_cluster.rs

```diff
@@ -47,103 +48,163 @@ impl InClusterJanusPair {
+        // harmless. The leader aggregator is paired as a *global* aggregator using the admin
+        // endpoint. We do this for two reasons:
+        //
+        // - setting up tasks with one global aggregator and one per-account aggregator is most
+        //   representative of the subscriber use cases Divvi Up supports,
+        // - pair a global aggregator implictly marks it as "first-party" in divviup-api, which is
+        //   necessary for the task we later provision to pass a validity check.
+        let paired_leader_aggregator = divviup_api
+            .pair_global_aggregator(&NewAggregatorRequest {
+                role: "Leader".to_string(),
+                name: "leader".to_string(),
+                api_url: Self::in_cluster_aggregator_api_url(&leader_namespace).to_string(),
+                dap_url: Self::in_cluster_aggregator_dap_url(&leader_namespace).to_string(),
+                bearer_token: leader_aggregator_api_auth_token,
+            })
+            .await;
+
+        let paired_helper_aggregator = divviup_api
+            .pair_aggregator(
+                &account,
+                &NewAggregatorRequest {
+                    role: "Helper".to_string(),
+                    name: "helper".to_string(),
+                    api_url: Self::in_cluster_aggregator_api_url(&helper_namespace).to_string(),
+                    dap_url: Self::in_cluster_aggregator_dap_url(&helper_namespace).to_string(),
+                    bearer_token: helper_aggregator_api_auth_token,
+                },
+            )
+            .await;
+
+        let provision_task_request = NewTaskRequest {
+            name: "Integration test task".to_string(),
+            leader_aggregator_id: paired_leader_aggregator.id,
+            helper_aggregator_id: paired_helper_aggregator.id,
+            vdaf: task.vdaf().try_into().unwrap(),
+            min_batch_size: task.min_batch_size(),
+            max_batch_size: match task.query_type() {
+                QueryType::TimeInterval => None,
+                QueryType::FixedSize { max_batch_size } => Some(*max_batch_size),
+            },
+            expiration: "3000-01-01T00:00:00Z".to_owned(),
+            time_precision_seconds: task.time_precision().as_seconds(),
+            hpke_config: STANDARD.encode(task.collector_hpke_config().get_encoded()),
         };
-        let helper_task = helper_task
-            .with_aggregator_endpoints(helper_endpoints)
+
+        // Provision the task into both aggregators via divviup-api
+        let provisioned_task = divviup_api
+            .create_task(&account, &provision_task_request)
+            .await;
+
+        let collector_auth_tokens = divviup_api
+            .list_collector_auth_tokens(&provisioned_task)
+            .await;
+
+        // Update the task with the ID and collector auth token from divviup-api, and aggregator
+        // endpoints with the in-cluster URLs.
```

Comment:
This is partly outdated, as the endpoints aren't updated here anymore. `submit_measurements_and_verify_aggregate()` already overrides the host and port of aggregator endpoints to be 127.0.0.1 and whatever the local forwarded port is.
```suggestion
        // Update the task with the ID and collector auth token from divviup-api.
```